Peer/Peer.csproj

```diff
@@ -27,26 +27,23 @@
     <Authors>Thomas Showers, Chris Nantau, Peer Contributors</Authors>
     <Company>wareismymind</Company>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Description> A CLI dashboard for visualizing active pull requests from Github and more!</Description>
```

Comment:
```suggestion
    <Description> A CLI dashboard for active pull requests from Github and more!</Description>
```

Reply:
I take "visualizing" to mean creating a visual representation that facilitates understanding the information in a way that simply _having_ the information doesn't. This is more just showing the information that ShitHub fails to show you. f7m